utilities/helpers.js

```diff
@@ -28,17 +28,15 @@ export const copyToClipBoard = async (content) => {
     })
 }
 
-// toast configuration
-export const toastGlobalConfig = (type) => {
-
+// global function for getting and overridding toast notification configuation 
+export const toastGlobalConfig = (optionsObj) => {
+  const options = optionsObj ? optionsObj : {}
```

Comment:
Instead of assigning an empty object to `options`  and then checking every property of the `options` object later, this could be simplified to set the default values (those if `optionsObj` is falsy). If the code were modified to maintain type-expectations (such that this function could reliably expect an object; e.g. reduce the prevalence of `null`), the spread operator could be introduced to clean this up even further.

Reply:
that was really cool. thanks @allie-drake